sql/core/src/main/scala/org/apache/spark/sql/execution/LocalTableScanExec.scala

```diff
@@ -30,6 +30,8 @@ case class LocalTableScanExec(
     output: Seq[Attribute],
     @transient rows: Seq[InternalRow]) extends LeafExecNode {
 
+  override val nodeName: String = s"Scan LocalTable ${output.map(_.name).mkString("[", ",", "]")}"
```

Comment:
It sounds like we have duplicate info about `output` in `stringArgs`.

Reply:
I believe you are referring to the duplication at : 
https://github.com/apache/spark/blob/3f958a99921d149fb9fdf7ba7e78957afdad1405/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala#L466

```
def simpleString: String = s"$nodeName $argString".trim
```

Am changing this line to just have `Scan LocalTable`